doc/source/installation.rst

```diff
@@ -126,10 +126,13 @@ the ``-d`` or ``--dir=`` argument.
 6. Include in application
 .........................
 
-To include ``pyramid_oereb`` into your existing Pyramid application, you first have to include the plugin in
-your application's main method.
+In your ``setup.py`` you shouldn't directly depends on ``pyramid_oereb`` because you will not depends on any
+other package. You should use ``pyramid_oereb[recommend]`` to gets the recommended versions of the
```

Comment:
... "to get the recommended versions of the dependencies, or ``pyramid_oereb[no-version]`` to get the dependencies without specifying which versions of these dependencies shall be used."

Reply:
Done :-)